defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Please add a new line between `---` and the vars. Also please comment and describe what this variable does and how to use it.